tests/foreman/installer/test_infoblox.py

```diff
@@ -0,0 +1,153 @@
+"""Tests for Infoblox Plugin
+
+@Requirement: Infoblox
+
+@CaseAutomation: Automated
+
+@CaseLevel: Acceptance
+
+@CaseComponent: Installer
+
+@TestType: Functional
+
+@CaseImportance: High
+
+@Upstream: No
+"""
+
+from robottelo.decorators import stubbed, tier3
+from robottelo.test import TestCase
+
+
+class InfobloxTestCase(TestCase):
+    @stubbed()
+    @tier3
+    def test_update_appliance_credentials(self):
+        """Check infoblox appliance credentials are updated
+
+        @id: 2e84a8b4-79b6-11e6-8bf8-68f72889dc7f
+
+        @Steps:
+
+        1. Pass appliance credentions via installer
+
+        @Assert: config/dns_infoblox.yml should be updated with
+        infoblox_hostname, username & password
+
+        @caseautomation: notautomated
+        """
+
+    @stubbed()
+    @tier3
+    def test_enable_dns_plugin(self):
+        """Check Infoblox DNS plugin can be enabled on host
+
+        @id: f8be8c34-79b2-11e6-8992-68f72889dc7f
+
+        @Steps:
+
+        1. Enable Infoblox plugin via installer
```

Comment:
I think we can define one liner step along with @Steps: tag ?
`@Steps: Enable Infoblox plugin via installer`